blue-xdma/src/XDMATestbench.bsv

```diff
@@ -8,22 +8,23 @@ import AXI4Slave :: *;
 import XDMADescriptorGenerator :: *;
 import StmtFSM :: *;
 import Counter :: *;
+import BRAMFIFO::*;
```

Comment:
`BRAMFIFOF :: *;`

Reply:
In the `BRAMFIFO.bsv` file, the `mkSizedBRAMFIFOF` is exported, but the `BRAMFIFOF.bsv` file does not exist. Replacing it with `BRAMFIFOF :: *;` would cause some issues.

You can find the relevant code at this link: https://github.com/B-Lang-org/bsc/blob/0eb551dcdb2fad5dd455d5937519b768ddb61fd4/src/Libraries/Base3-Misc/BRAMFIFO.bsv#L23.